dotnet/tests/Microsoft.Agents.AI.Hosting.AzureFunctions.IntegrationTests/SamplesValidation.cs

```diff
@@ -635,10 +635,14 @@ private Process StartFunctionApp(string samplePath, List<OutputLog> logs)
             RedirectStandardError = true,
         };
 
-        string openAiEndpoint = s_configuration["AZURE_OPENAI_ENDPOINT"]
-            ?? throw new InvalidOperationException("AZURE_OPENAI_ENDPOINT is not set in the environment variables or user secrets");
-        string openAiDeployment = s_configuration["AZURE_OPENAI_DEPLOYMENT"]
-            ?? throw new InvalidOperationException("AZURE_OPENAI_DEPLOYMENT is not set in the environment variables or user secrets");
+        string openAiEndpoint =
```

Comment:
Do we need any update in the samples main? ex; https://github.com/microsoft/agent-framework/blob/304b80965575cddd50aa41a403292f784a38e7ec/dotnet/samples/AzureFunctions/01_SingleAgent/Program.cs#L13